src/app/modules/gb-cohort-selection-module/constraint-components/gb-constraint/gb-constraint.component.ts

```diff
@@ -32,7 +32,7 @@ export class GbConstraintComponent implements OnInit {
               protected treeNodeService: TreeNodeService,
               protected resourceService: ResourceService,
               protected constraintService: ConstraintService,
-              protected queryService: QueryService,
+              protected queryService: CohortService,
```

Comment:
Here it could be also renamed: 
```suggestion
              protected cohortService: CohortService,
```